src/ControllerV2/BeneficiaryResetPasswordController.php

```diff
@@ -17,7 +20,7 @@
 class BeneficiaryResetPasswordController extends AbstractController
 {
     #[IsGranted('UPDATE', 'beneficiary')]
-    #[Route(path: '/beneficiary/{id}/reset-password', name: 'reset_password_beneficiary', methods: ['GET'])]
+    #[Route(path: '/beneficiary/{id<\d+>}/reset-password', name: 'reset_password_beneficiary', methods: ['GET'])]
```

Comment:
We could share this path `/beneficiary`, il would also tend to use `/beneficiaries`, feels more restfull